### DISCUSSION TITLE
百度网盘，为什么我用RaiDrive挂载，下载文件出来的速度跑不满

### DISCUSSION THREAD

**Discussion by WANGDADADA910930** [AlistGo/alist#5623] [Q&A]
![@BLJ65 4GPW$6{7~1VR6}(7](https://github.com/alist-org/alist/assets/128799274/e9988687-09eb-48ef-b734-a6543d4f72f3)
![YC8P N$1)_T%2~)2KT{`)EN](https://github.com/alist-org/alist/assets/128799274/48690205-3350-4a2b-8dbe-888caa2f6c6f)

**Comment by zhoutao-cn** [comment]
Alist 从前天晚上起被限速了。

**Reply by zhoutao-cn** [reply]
但是你这个速度可以啊！我们都是2M以下的。

**Reply by WANGDADADA910930** [reply]
webdav的本地挂载是被限速了吗？>_<

**Reply by WANGDADADA910930** [reply]
我是有百度网盘会员的，用这个挂载只是想单纯的管理我自己本地的一些.ai .psd之类的文件，在百度客户端里显示不了缩略图

**Reply by zhoutao-cn** [reply]
是我误会了。是 阿里云盘 对 Alist 进行限速了。
